pkg/daemon/discover/discover.go

```diff
@@ -398,11 +398,16 @@ func updateDeviceCM(context *clusterd.Context) error {
 func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 	devices := make([]sys.LocalDisk, 0)
 	localDevices, err := clusterd.DiscoverDevices(context.Executor)
-	logger.Infof("localdevices: %+v", localDevices)
 	if err != nil {
 		return devices, fmt.Errorf("failed initial hardware discovery. %+v", err)
 	}
 
+	var devicesList []string
+	for _, device := range localDevices {
+		devicesList = append(devicesList, fmt.Sprintf("%+v", device))
+	}
+	logger.Infof("localdevices: %q", strings.Join(devicesList, ", "))
```

Comment:
This is going to be verbose information.
- Should this be debug level instead of info?
- I know I suggested the concat to a single string, but now it seems better to log each one on a different line since each device info will be so verbose.
- The ceph operator doesn't actually use this info for creating OSDs. The discovery in the osd prepare pod is more important for creating OSDs.

Reply:
@travisn What about "splitting" the log? Just the device names as info and debug all the other information?
I personally feel like this line of information (what devices were detected) can be pretty useful.

Reply:
I was also thinking of that possibility. On a single line the info log could show a comma-separated list of devices, then the debug log could show one line per device with all the details.